lib/active_merchant/billing/gateways/checkout_v2.rb

```diff
@@ -459,28 +457,30 @@ def access_token_url
       end
 
       def setup_access_token
-        request = 'grant_type=client_credentials'
-        begin
-          raw_response = ssl_post(access_token_url, request, access_token_header)
-        rescue ResponseError => e
-          raise OAuthResponseError.new(e)
-        else
-          response = parse(raw_response)
+        response = parse(ssl_post(access_token_url, 'grant_type=client_credentials', access_token_header))
+        @options[:access_token] = response['access_token']
+        @options[:expires] = (DateTime.now + response.fetch('expires_in', 0).seconds).strftime('%Q').to_i
```

Comment:
Some thing for this fetch it needs to be updated since it's not going to default to 0

Reply:
ok

Reply:
done!